Extension/src/testHook.ts

```diff
@@ -4,25 +4,33 @@
  * ------------------------------------------------------------------------------------------ */
 'use strict';
 
-import { CppToolsTestHook, Status } from 'vscode-cpptools/out/testApi';
+import { CppToolsTestHook, Status, IntelliSenseStatus } from 'vscode-cpptools/out/testApi';
 import * as vscode from 'vscode';
 
 export class TestHook implements CppToolsTestHook {
+    private intelliSenseStatusChangedEvent: vscode.EventEmitter<IntelliSenseStatus> = new vscode.EventEmitter<IntelliSenseStatus>();
     private statusChangedEvent: vscode.EventEmitter<Status> = new vscode.EventEmitter<Status>();
 
+    // The StatusChanged event is deprecated in CppToolsTestHook API.
     public get StatusChanged(): vscode.Event<Status> {
         return this.statusChangedEvent.event;
     }
 
+    public get IntelliSenseStatusChanged(): vscode.Event<IntelliSenseStatus> {
+        return this.intelliSenseStatusChangedEvent.event;
+    }
+
     public get valid(): boolean {
-        return !!this.statusChangedEvent;
+        return !!this.intelliSenseStatusChangedEvent;
     }
 
-    public updateStatus(status: Status): void {
-        this.statusChangedEvent.fire(status);
+    public updateStatus(status: IntelliSenseStatus): void {
+        this.intelliSenseStatusChangedEvent.fire(status);
```

Comment:
@michelleangela 
Note, that you need to fire both events here or else the old one will be totally broken.  I don't know if anyone took a dependency on this or not, but it's best not to break this.